development/language/plurals.rst

```diff
@@ -0,0 +1,203 @@
+* 1 - ends in 01: 1, 101, 201, ...
+* 2 - ends in 02: 2, 102, 202, ...
+* 3 - ends in 03-04: 3, 4, 103, 104, 203, 204, ...
+* 4 - everything else: 0, 5, 6, 7, 8, 9, 10, 11, ...
+
+Rule #11
+--------
+* Families: Celtic (Irish Gaeilge)
+* 1 - 1
+* 2 - 2
+* 3 - is 3-6: 3, 4, 5, 6
+* 4 - is 7-10: 7, 8, 9, 10
+* 5 - everything else: 0, 11, 12, ...
+
+Rule #12
+--------
+* Families: Semitic (Arabic)
+* 1 - 1
+* 2 - 2
+* 3 - ends in 03-10: 3, 4, ... 10, 103, 104, ... 110, 203, 204, ...
+* 4 - ends in 11-99: 11, ... 99, 111, 112, ...
+* 5 - everything else: 100, 101, 102, 200, 201, 202, ...
+* 6 - 0
+
+Rule #13
+--------
+* Families: Semitic (Maltese)
+* 1 - 1
+* 2 - ends in 01-10: 0, 2, 3, ... 9, 10, 101, 102, ...
+* 3 - ends in 11-19: 11, 12, ... 18, 19, 111, 112, ...
+* 4 - everything else: 20, 21, ...
+
+Rule #14
+--------
+* Families: Slavic (Macedonian)
+* 1 - ends in 1: 1, 11, 21, ...
+* 2 - ends in 2: 2, 12, 22, ...
+* 3 - everything else: 0, 3, 4, ... 10, 13, 14, ... 20, 23, ...
+
+Rule #15
+--------
+* Families: Icelandic
+* 1 - ends in 1, not 11: 1, 21, 31, ... 101, 121, 131, ...
+* 2 - everything else: 0, 2, 3, ... 10, 11, 12, ... 20, 22, ...
+
+How to use the rules
+====================
+The first thing your language package needs, is a definition, which rule to use for your package. This is done in the ``language/xy/common.php`` language file at the beginning of the array, (Rule #1 is the rule for the English language and will be used as default, if you don't specify one):
+
+.. code-block:: php
+
+    'PLURAL_RULE' => 1,
+
+I'm now using rule **#13** as example:
+
+It has the following rows:
+
+* 1 - 1
+* 2 - ends in 01-10: 2, 3, ... 9, 10, 101, 102, ...
+* 3 - ends in 11-19: 11, 12, ... 18, 19, 111, 112, ...
+* 4 - everything else: 20, 21, ...
+
+While the English language only has 3 rows in its array:
+
+.. code-block:: php
+
+    'EXAMPLE' => array(
+        1 => '1 example',
+        2 => '2 or more examples',
+    ),
+You need to specify the zero-row and 4 rows for the "plurals":
+
+.. code-block:: php
+
+    'EXAMPLE' => array(
+        1 => '1 example',
+        2 => '[0 or number ending with 01-10] examples',
+        3 => '[number ending with 11-19] example',
+        4 => 'even more examples',
+    ),
+If you want to get a separate handling for 0, you can simple add the 0-case:
+
+.. code-block:: php
+
+    'EXAMPLE' => array(
+        0 => 'No example',
+        1 => '1 example',
+        2 => '[zero is not handled here anymore! Only number ending with 01-10] examples',
+        3 => '[number ending with 11-19] example',
+        4 => 'even more examples',
+    ),
+
+If you forget a line the system will automatically use the row before. So if you forget to add the *3*-row, it will use *2*-row for 11-19 as well. If there is no previous row, it uses the last row of the array.
+
+
+**Ensure that your cases are ordered ascending**, otherwise the system might act weird, if not all keys are present.
```

Comment:
"Ensure your cases are in ascending order, otherwise the system may produce unexpected results if any keys are missing or out of order."